test/end-to-end/vehicle.js

```diff
@@ -374,6 +374,69 @@ test('vehicle batch', async(t) => {
   t.is(location.meta.requestId.length, 36);
 });
 
+test('vehicle request - odometer', async(t) => {
+  const response = await t.context.volt.request(
+    'get',
+    'odometer',
+    {},
+    {
+      'sc-unit-system': 'imperial',
+    }
+  );
+  t.deepEqual(
+    _.xor(_.keys(response), [
+      'distance',
+      'meta',
+    ]),
+    [],
+  );
+
+  t.truthy(typeof response.distance === 'number');
+  t.truthy(response.meta.dataAge instanceof Date);
+  t.is(response.meta.requestId.length, 36);
+  t.is(response.meta.unitSystem, 'imperial');
+});
+
+test('vehicle request - lock', async(t) => {
```

Comment:
I would avoid running a lock request in the tests, we can run any other request we want here. Might be good to eventually test a brand-specific endpoint for this test.